target/linux/ath79/image/generic.mk

```diff
@@ -160,6 +181,18 @@ define Device/avm_fritz300e
 endef
 TARGET_DEVICES += avm_fritz300e
 
+define Device/avm_fritz3390_wasp
```

Comment:
That would have to be avm_fritz3390-wasp.
Despite, I wonder why we have abbreviated names again on the new ath79 target.
Actually, this should be avm_fritzbox-3390-wasp. I might change that for the other devices, so be prepared.